File: src/main/java/com/lczq/dbquery/DBQueryApplication.java
package com.lczq.dbquery;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@MapperScan("com.lczq.dbquery.mapper")
@SpringBootApplication
public class DBQueryApplication
{
    public static void main(String[] args)
    {
        SpringApplication.run(DBQueryApplication.class, args);
    }
}

File: src/main/kotlin/com/github/wgzhao/dbquery/DBQueryApplication.kt
package com.github.wgzhao.dbquery

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
object DBQueryApplication {
    @JvmStatic
    fun main(args: Array<String>) {
        SpringApplication.run(DBQueryApplication::class.java, *args)
    }
}
